### DISCUSSION TITLE
Cell tradings of Daly BMS

### DISCUSSION THREAD

**Discussion by DIVERSDIVERS** [Louisvdw/dbus-serialbattery#511] [Q&A] [answered]
diver support for Daly BMS to track each cell

**Comment by rbeudel** [comment]
yes

**Reply by DIVERSDIVERS** [reply]
![Bild_2023-04-07_094559322](https://user-images.githubusercontent.com/129951689/230565851-ce7be3f0-06e6-4d11-bff3-02c51e13e036.png)

**Comment by rbeudel** [comment]
![Screenshot (11)](https://user-images.githubusercontent.com/97027663/230568432-85cf2975-7110-4662-b12d-cfa2779e1f7b.png)
Do you mean this? I dont understand the problem, but my english is very bad.

**Reply by DIVERSDIVERS** [reply]
yes, i mean this how you did it?

**Reply by mr-manuel** [reply]
Probably a different Daly firmware/hardware revisions. @rbeudel @DIVERSDIVERS what are your hardware/firmware versions of the Daly BMS and when you bought it?

**Reply by rbeudel** [reply, marked as answer]
My hardware is BMS-ST103-309E. It is one year old and the firmware is DalyBmsApp_103_309(11_221213_INVT).  May it be the RS485 Problem discibed here? [https://github.com/Louisvdw/dbus-serialbattery/issues/397](url)

**Comment by mr-manuel** [comment]
> Please open an issue for that. Since I have no Daly BMS I unfortunately cannot help you further.
> Please post driver version, Venus OS Version, BMS connection type, BMS make and BMS hard/firmware version (found in the BMS app).

As I wrote in https://github.com/Louisvdw/dbus-serialbattery/discussions/509#discussioncomment-5541958 please open a [new issue](https://github.com/Louisvdw/dbus-serialbattery/issues/new?assignees=&labels=&template=bug_report.md&title=) for that with all the needed informations (see quote above). This is a question under discussions. Please close it once you opened the issue.

**Comment by DIVERSDIVERS** [comment]
Sorry to bother you again, unfortunately it doesn't update the cell voltage
![001](https://user-images.githubusercontent.com/129951689/230611904-dc8be80e-9167-4f7e-bba2-85629a05de72.JPG)
![002](https://user-images.githubusercontent.com/129951689/230611907-22583f07-411b-4f6c-8606-47c6f67e1258.JPG)

**Reply by rbeudel** [reply]
I did not change all the parameters like in fix #397. It runs with 16 cells, but it seems you have to do all the changes. @towatai wrote, his 8 cells installation runs with the fixes.
